Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -683,6 +697,12 @@ class PlaceDetailViewController: UIViewController {
         filterVC.delegate = self
         filterVC.placeCount = dataSource.numberOfPlaces()
         present(filterVC, animated: true, completion: nil)
+
+        UIView.animate(withDuration: cardSlideDuration) {
```

Comment:
nit: could be a helper method, `slideCurrentCardView(willBeShown:)`. That way, there's no code duplication (between hide & show) and it's a little more readable.